turbopack/packages/devlow-bench/src/interfaces/snowflake.ts

```diff
@@ -0,0 +1,117 @@
+import type { Interface } from "../index.js";
+import os from "os";
+import { CPU_ARCH, CPU_MODEL, GIT_BRANCH, GIT_SHA, IS_CI, NODE_VERSION, NUM_CPUS, OS, OS_RELEASE, USERNAME } from "./constants.js";
+import { randomUUID } from "crypto";
+
+type DevlowMetric = {
+  event_time: number;
+  scenario: string;
+  props: Record<string, string | number | boolean | null>;
+  metric: string;
+  value: number;
+  unit: string;
+  relative_to?: string;
+  is_ci: boolean;
+  os: string;
+  os_release: string;
+  cpus: number;
+  cpu_model: string;
+  user: string;
+  arch: string;
+  total_memory: number;
```

Comment:
It's probably too late to change this? But it'd be nice if this field name contained units.